redhat-cop/infra-ansible
# Add initial AWS key pair management

What does this PR do?

Allows for creation/management of AWS keys-pairs

How should this be tested?

Update sample inventory to use valid aws credentials and the keypair(s) of your choice. Then run the playbook `keypairs.yml` underneath tests. You should then be able to run it / rerun it to show that it creates the network.

Is there a relevant Issue open for this?

N/A

Other Relevant info, PRs, etc.

People to notify

cc: @redhat-cop/infra-ansible
